packages/unchained-client/src/evm/bnbsmartchain/parser/index.ts

```diff
@@ -2,13 +2,19 @@ import { bscAssetId } from '@shapeshiftoss/caip'
 
 import { Tx } from '../../../generated/bnbsmartchain'
 import { BaseTransactionParser, TransactionParserArgs } from '../../parser'
+import * as zrx from '../../parser/zrx'
 import * as bep20 from './bep20'
 
+export const ZRX_BSC_PROXY_CONTRACT = '0xDef1C0ded9bec7F1a1670819833240f027b25EfF'
```

Comment:
To be consistent with my previous comments
```suggestion
export const ZRX_BSC_PROXY_CONTRACT_ADDRESS = '0xDef1C0ded9bec7F1a1670819833240f027b25EfF'
```

Reply:
All of the parsers use this same terminology and match with `new zrx.Parser({ proxyContract: ZRX_BSC_PROXY_CONTRACT })` proxyContract param. I think it is clear enough at this time